broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -1025,8 +1026,15 @@ private RemotingCommand queryTopicConsumeByWho(ChannelHandlerContext ctx,
             groups.addAll(groupInOffset);
         }
 
+        HashSet<String> filteredGroups = new HashSet<String>();
```

Comment:
It would be better to make an optional configuration here, because some users also want to show the consumption progress of topics that they have subscribed to, and it seems that users in the community are already used to this usage, so forward compatibility is necessary

Reply:
@duhenglucky so maybe I should add request flag for client, then broker will check flag bool indicator to dicide to filter ?

Reply:
@ IMHO, it seems that add a flag in the client will be a better choice